From: Tiago Gaspar <tiagogaspar8@gmail.com>
Date: Fri, 21 Jan 2022 13:39:30 +0000
Subject: [PATCH] libtool: Reduce the required version

Current Openwrt's libtool verion is 2.4.2, yet, jool had the requirement
set to 2.4.6 in source.
Change that to Openwrt's version.

Signed-off-by: Tiago Gaspar <tiagogaspar8@gmail.com>

--- a/configure.ac
+++ b/configure.ac
@@ -5,7 +5,7 @@ AC_PREREQ([2.68])
 AC_INIT([Jool], [4.1.6], [jool@nic.mx])
 AC_CONFIG_SRCDIR([src/common/xlat.h])
 AM_INIT_AUTOMAKE([subdir-objects])
-LT_PREREQ([2.4.6])
+LT_PREREQ([2.4.2])
 
 # Checks for programs.
 AC_PROG_CC
